lib/public/views/LhcFills/Overview/LhcFillsOverviewModel.js

```diff
@@ -28,7 +29,13 @@ export class LhcFillsOverviewModel extends OverviewPageModel {
     constructor(stableBeamsOnly = false) {
         super();
 
+        this._filteringModel = new FilteringModel({});
```

Comment:
I believe here we should do the other way around. Instead of keeping a hardcoded filter (stableBeamsOnly), we should be integrating that into the FilteringModel.

Have a look at `RunOverviewModel` and its implementation for the very similar switchInput of `PHYSICS ONLY`.